assets/training/finetune_acft_image/src/finetune/finetune.py

```diff
@@ -693,16 +693,12 @@ def main():
             Setting label_smoothing_factor to 0.0 from {args.label_smoothing_factor}"
```

Comment:
Can we already bump up the component version, env version in the component and component version in the pipeline components?

Reply:
Sure. All classification related components are bumped up.

Reply:
Updated component versions